sale_order_field_advanced/models/product_category.py

```diff
@@ -0,0 +1,10 @@
+# © 2025 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+
+from odoo import models, fields
+
+
+class ProductCategory(models.Model):
+    _inherit = 'product.category'
+
+    has_process_control = fields.Boolean(string='Has Process Control', default=False)
```

Comment:
When field name definition perfectly matches desired label, string attribute should be omitted:
```suggestion
    has_process_control = fields.Boolean(default=False)
```